### PR TITLE
Fix import questions

### LINKED ISSUE #235: jak importujemy pytania z błędnym json to importuje to jako puste pytania
Testowanie czy JSON jest prawidłową listą pytań, Odrzucić wszystkie błędne pytania.
Powiadomienie typu [Snackbar](https://material-ui.com/components/snackbars/) czy udało się zaimportować i ile pytań
Tak samo jak w #234 jeśli importowane pytania mają tą samą nazwę, należy im te nazwy zmienić dodając "(x)" do nazwy, gdzie x to liczba kolejnej powtórki

### PR DESCRIPTION
**Pull request by Karolk99**
image

### CODE REVIEW

**Comment by Karolk99** [comment]
it is really easy to change the position of snackbar so don't be bothered

**Review by michal367 — CHANGES_REQUESTED** [review]
Questions with the same title should not be imported
Snackbar should be lower
Snackbar should dissapear after some time
Change "Importowano" to "Zaimportowano"
App breaks if I import empty file or json with different structure (e.g. {}) or file is not json

**Review by Derstilon — COMMENTED** [review]
Let me know when you will be done with @michal367's changes

File: src/lecturer/components/questionsListView/QuestionsListView.tsx [hunk 1]
```diff
@@ -144,9 +204,22 @@ export function QuestionsListView() {
...
     return (
+        <>
+        <Snackbar
+            anchorOrigin={{ vertical, horizontal }}
+            open={open}
+            onClose={handleClose}
+            message={message}
+            key={vertical + horizontal}
+            autoHideDuration={1000}
+        />
```
Comment (michal367) [inline src/lecturer/components/questionsListView/QuestionsListView.tsx]:
```suggestion
        <Snackbar
            anchorOrigin={{ vertical, horizontal }}
            open={open}
            onClose={handleClose}
            message={message}
            key={vertical + horizontal}
            autoHideDuration={1000}
            style={{top: "60px"}}
        />
```

**Review by michal367 — CHANGES_REQUESTED** [review]

**Review by michal367 — APPROVED** [review]

**Review by Derstilon — APPROVED** [review]
Good job. Looks clean

**Pull request merged**